cloud/google/gce.py

```diff
@@ -145,6 +145,13 @@
       - if set boot disk will be removed after instance destruction
     required: false
     default: "true"
+  preemptible:
+    version_added: "2.1"
+    description:
+      - if set to true, instances will be preemptible and time-limited.
+        (requires libcloud >= 0.20.0)
```

Comment:
@curious-eyes - I think Brian was also suggesting you also update the `requirements` section of the module docstring to suggest >= 0.20.0 is preferred for all features.

I think that's the final tweak and then you're all set.